.github/workflows/Java-18-ea-Instrumentation-Tests.yml

```diff
@@ -15,7 +15,7 @@ on:
 
 jobs:
   tests:
-    uses: newrelic/newrelic-java-agent/.github/workflows/X-Reusable-Test.yml@main
+    uses: newrelic/newrelic-java-agent/.github/workflows/X-Reusable-Test.yml@java-18-ea
```

Comment:
Unfortunately, this is right for now due to reusable workflow.  I tried to reference one in THIS branch and didn't work.